### PR TITLE
Fix comment about mdcb/hybrid

### PR DESCRIPTION
**Pull request by letzya**

### CODE REVIEW

**Comment by asoorm** [comment]
I haven't looked at the code - but is this true? 

The gateways write analytics to Redis - then pull the analytics periodically to send to MDCB component?

I thought the original version was correct

**Comment by letzya** [comment]
Per @buger

File: tyk-docs/content/analyse/redis-mongodb-sizing.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The average single request analytics record (without detailed logging turned on)
 
 In terms of Redis, in addition to key storage itself, it should be able to hold the last 10 seconds of analytics data, preferably more, in case of a Tyk pump failure. So if you have 100 requests per second, you will need approximately 6MB for storing 60 seconds of data. Be aware that if detailed logging is turned on, this can grow by a magnitude of 10. 
 
-> MDCB and Hybrid clients, instead of writing data to a temporary Redis list, sends it directly to the MDCB server, which processes them similar to Pump.
+> MDCB and Hybrid clients - the gateways write the data to a temporary Redis list and periodically send the anlytics directly to the MDCB server, which processes them similar to Pump.
```
Comment (joshblakeley) [inline tyk-docs/content/analyse/redis-mongodb-sizing.md]: ...which processes them for purging to MongoDB

Something like that

Comment (joshblakeley) [reply tyk-docs/content/analyse/redis-mongodb-sizing.md]: Also typo on `analytics`

**Comment by marksou** [comment]
@buger can you reply to @asoorm comment?

**Comment by buger** [comment]
@asoorm yes this is how it works now, logic was changed. It was made to improve failover of Hybrid gateways, so if MDCB goes down for short period of time, you do not lost analytics. 

So the only difference at the moment that Hybrid gateways "push" to MDCB though Redis, and On-Premise gateways just write to Redis, and Pump "pulls" data from Redis.

**Comment by marksou** [comment]
@buger Are you happy for me to merge?

**Comment by buger** [comment]
👌

**Pull request merged**